README.md

```diff
@@ -14,14 +14,30 @@ $ openssl rsa -in keys/private.pem -outform PEM -pubout -out keys/public.pem
 
 ```clj
 ;; Fetching user account on remote server
-(as-> "@jahfer@mastodon.social" $
-  (activitypub/parse-account $)
-  (map $ [:domain :username])
-  (apply webfinger/fetch-user-id $)
-  (activitypub/fetch-user $)
-  (select-keys $ ["followers" "following" "inbox" "outbox"
-                  "endpoints" "publicKey" "summary" "attachment"
-                  "name" "preferredUsername" "icon" "published"]))
+(require '[clj-activitypub.core :as activitypub])
+(require '[clj-activitypub.webfinger :as webfinger])
+(require '[clojure.walk :refer [keywordize-keys]])
+(require '[clojure.pprint :refer [pprint]])
+
+;;; Use any ActivityPub account handle you like - for example, your own
+(def account-handle "@simon_brooke@mastodon.scot")
```

Comment:
I'll be a nitpick here and say that I prefer to use my own handle in the README, as was done previously (i.e. `@jahfer@mastodon.social`).